src/JJMasterData.Core/DataManager/FormManager.cs

```diff
@@ -35,34 +30,27 @@ public Hashtable UserValues
         }
     }
 
-    /// <summary>
-    /// Objeto responsável por fazer toda a comunicação com o banco de dados
-    /// </summary>
+    /// <inheritdoc cref="Commons.Dao.DataAccess"/>
     public IDataAccess DataAccess
     {
         get => _dataAccess ??= JJService.DataAccess;
         set => _dataAccess = value;
     }
 
-    /// <summary>
-    /// Objeto responsável por parsear expressoões
-    /// </summary>
-    public ExpressionManager Expression => _expression ??= new(UserValues, DataAccess);
-
-    /// <summary>
-    /// Configurações pré-definidas do formulário
-    /// </summary>
-    public FormElement FormElement { get; set; }
-
-    /// <summary>
-    /// Objeto responsável por traduzir o elemento base em comandos para o banco de dados
-    /// </summary>
+    /// <inheritdoc cref="Commons.Dao.Entity.Factory"/>
     public Factory Factory
     {
         get => _factory ??= new Factory(DataAccess);
         set => _factory = value;
     }
 
+    /// <inheritdoc cref="ExpressionManager"/>
+    public ExpressionManager Expression => _expression ??= new(UserValues, DataAccess);
+
+    /// <inheritdoc cref="FormElement"/>
+    public FormElement FormElement { get; set; }
+
+
     public FormManager(FormElement formElement)
     {
         FormElement = formElement;
```

Comment:
Add a 
```cs
if(FormElement == null)
    throw new ArgumentNullException(nameof(FormElement))
```
So you don't need to check if the FormElement is null in the entire class